packages/cli/package.json

```diff
@@ -46,6 +46,7 @@
     "@oclif/plugin-not-found": "3.2.51",
     "@oclif/plugin-version": "2.2.28",
     "adm-zip": "0.5.16",
+    "decompress": "4.2.1",
```

Comment:
`decompress` was originally a dev dependency. I moved it to the `dependencies` section because now we're using it to extract build.zip in the "Testing build" step. The existing `adm-zip` package doesn't support extracting symlinks in a zip file, so I use `decompress` instead.